kahuna/conf/routes

```diff
@@ -17,7 +17,6 @@ GET     /assets/*file                                 controllers.Assets.version
 GET     /management/healthcheck                       com.gu.mediaservice.lib.management.Management.healthCheck
 GET     /management/manifest                          com.gu.mediaservice.lib.management.Management.manifest
 GET     /management/whoAmI                            com.gu.mediaservice.lib.management.InnerServiceStatusCheckController.whoAmI(depth: Int)
-GET     /management/innerServiceStatusCheck           com.gu.mediaservice.lib.management.InnerServiceStatusCheckController.statusCheck(depth: Int)
```

Comment:
I presume this healthcheck runs directly on thrall now and is superfluous?

Reply:
yes exactly (see commit message https://github.com/guardian/grid/pull/3403/commits/82950b6f04a2352a221f6ee8937c89ac812fb290)
(and just to clarify it doesn't run on a schedule or anything - to avoid unnecessary load - its just part of pingler for local dev and can now be hit directly since thrall is visible)